board/opentrons/ot2/genimage-ot2.cfg

```diff
@@ -10,7 +10,8 @@ image boot.vfat {
       "rpi-firmware/fixup.dat",
       "rpi-firmware/start.elf",
       "rpi-firmware/overlays",
-      "zImage"
+      "u-boot.bin",
+      "boot.scr"
```

Comment:
Is `.scr` some convention (as opposed to `.sh`)?

Reply:
Yes. `boot.scr` (exactly) is the name specified by [distro boot](https://github.com/u-boot/u-boot/blob/master/doc/README.distro) for a boot script (it can be overridden by configs but the rpi config [does not](https://github.com/u-boot/u-boot/blob/63f7e3fca391a50a499fed828fe16325fdee45f3/configs/rpi_3_32b_defconfig#L6)